lib/msal-common/test/client/ClientTestUtils.ts

```diff
@@ -327,4 +328,182 @@ export class ClientTestUtils {
             },
         };
     }
+    
+    static async createTestClientConfigurationTelem(): Promise<ClientConfiguration> {
```

Comment:
I've made it less repetitive and relied more on the other functions, but the reason I made it its own function is because I'm calling that functionality in two different test suites.

Reply:
Can you update the above one to optionally take in the parts you need (apiId, protocolMode) and attach them if provided? Would like to keep all of this centralized if possible

Reply:
That sounds like a much better solution, just fixed